internal/chart/application_chart.go

```diff
@@ -53,6 +53,11 @@ type ingress struct {
 	Https []httpsEndpoint `json:"https"`
 }
 
+type GatewayService struct {
```

Comment:
It would be great to document `GatewayService`, at least the purpose of having it. It's also OK to name it `gatewayService` because there is no usage outside this package.

Reply:
Done.